lib/templateCache.js

```diff
@@ -3,21 +3,37 @@ var handlebars = require('gulp-handlebars')
 var wrap = require('gulp-wrap')
 var declare = require('gulp-declare')
 var concat = require('gulp-concat')
+var path = require('path')
+var diyHbsHelpers = require('diy-handlebars-helpers')
+var hbsHelpers = require('./hbs-helpers')
 
 var cache = ''
+var partials=''
+var helpers=''
+var opts = {}
 
 function TemplateCache() {}
 
 TemplateCache.prototype.recompile = function(cb) {
 	cache = ''
+	partials = ''
+	helpers = ''
 	return this.compile(cb)
 }
+TemplateCache.prototype.setHbs = function(hbs) {
+	opts.handlebars = hbs
```

Comment:
`opts` is not local to this class, setting this changes it for all instances of TemplateCache. How about using `this.hbsOptions = { handlebars: hbs }`?